fondat/aws/bedrock/resources/versions.py

```diff
@@ -0,0 +1,71 @@
+"""Resource for managing agent versions."""
+
+from collections.abc import Iterable
+from typing import Any, Mapping
+
+from fondat.aws.client import Config
+from fondat.pagination import Cursor
+from fondat.resource import resource
+from fondat.security import Policy
+
+from ..clients import agent_client
+from ..decorators import operation
+from ..pagination import decode_cursor, paginate
+
+
+@resource
+class VersionsResource:
```

Comment:
Why not use the same pattern used for other resources : `VersionResource` and `VersionsResource` ?